frontend/app/routing.js

```diff
@@ -79,7 +133,7 @@ angular.module('openproject')
       templateUrl: '/templates/work_packages.list.new.html'
     })
     .state('work-packages.list.details', {
-      url: '/{workPackageId:[0-9]+}?query_props',
+      url: '/details/{workPackageId:[0-9]+}?query_props',
```

Comment:
I think it is not working as you intended, then. 
In the dev and release branches, this is the url for opening a details pane (table + details pane opened with the overview tab): `/projects/:project_id/work_packages/:id/overview`. In this branch, it will yield an empty page. Instead one has to query with `/projects/:project_id/work_packages/details/:id/overview`.

When simply removing the `details` part everything seems to be working again. I am not an expert on the routing but I believe that the state is scoped somehow to only be available scoped under '/projects/:project_id'.